package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "@streamyard/typeorm",
   "private": true,
-  "version": "0.3.16-1",
+  "version": "0.3.16-2",
```

Comment:
Lock should be updated after this change right?

Reply:
WDYM?